packages/editor/playground/playground.tsx

```diff
@@ -94,35 +124,50 @@ const App: FC = () => {
     const initialContentId = params.get("contentid")
     const enableDevTools = params.get("devtools") !== "false"
 
-    const note = useMemo(() => {
-        let content = defaultContent
-        if (isString(initialContent)) {
-            content = initialContent
-        } else if (initialContentId === "long") {
-            content = longContent
-        } else if (initialContentId === "just-code") {
-            content = justCodeContent
-        }
+    const [hasUnsavedChanges, setHasUnsavedChanges] = useState(false)
 
+    if (isString(initialContent) && initialContent.length > 0) {
```

Comment:
Remove `initialContent.length > 0` because we are using http://localhost:3001/?content= to open an empty editor.